SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -19,9 +19,4491 @@
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Differential_Pressure_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:AirDeltaPressureSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Alarm;1",
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Alarm_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Demand_Setpoint;1",
+      "OutputDtmi": "dtmi:com:willowinc:AirFlowSetpoint;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Direction_Status;1",
+      "OutputDtmi": "dtmi:com:willowinc:State;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Loss_Alarm;1",
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Rate_Command;1",
+      "OutputDtmi": "dtmi:com:willowinc:Actuator;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Setpoint_Limit;1",
+      "OutputDtmi": "dtmi:com:willowinc:Limit;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Flow_Status;1",
+      "OutputDtmi": "dtmi:com:willowinc:State;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Grains_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:HumidityRatioSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Plenum;1",
+      "OutputDtmi": "dtmi:com:willowinc:AirPlenum;1",
+      "IsIgnored": true
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Quality_Sensing_Device;1",
+      "OutputDtmi": "dtmi:com:willowinc:AirQualitySensor;1",
+      "IsIgnored": true
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Static_Pressure_Step_Parameter;1",
+      "OutputDtmi": "dtmi:com:willowinc:Parameter;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_System;1",
+      "OutputDtmi": "dtmi:com:willowinc:HVACAirSystem;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Temperature_Alarm;1",
+      "OutputDtmi": "dtmi:com:willowinc:AlarmSensor;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Temperature_Integral_Time_Parameter;1",
+      "OutputDtmi": "dtmi:com:willowinc:Parameter;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Temperature_Setpoint_Limit;1",
+      "OutputDtmi": "dtmi:com:willowinc:Limit;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Temperature_Step_Parameter;1",
+      "OutputDtmi": "dtmi:com:willowinc:Parameter;1"
+    },
+    {
+      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Wet_Bulb_Temperature_Sensor;1",
+      "OutputDtmi": "dtmi:com:willowinc:Wet_Bulb_Temperature_Sensor;1"
```

Comment:
Was this intentional to underscore the output? Willow has other WetBulbTemperatureSensors without underscores